src/app/components/CentersOfExcellence/CentersOfExcellence.tsx

```diff
@@ -61,14 +62,17 @@ const Center = () => {
     }
     return htmlArr;
   };
+  const noResult = ()=>{
+    return (
+      <NoResults flag={3}/>
+    );
+};
 
   return (
-    <div className="container">
-      <h1 className="font-bold text-5xl text-gray-700 text-center mt-2 mb-2 tracking-widest">
-        Centers of Excellence
-      </h1>
-      <div className="container md:mx-auto md:px-20 md:py-5 m-auto grid grid-flow-row lg:grid-cols-3 md:grid-cols-2 sm:grid-cols-1">
-        {isCoesLoaded ? getCenters() : null}
+    <div className='container'>
+      <h1 className='font-bold text-5xl text-gray-700 text-center mt-2 mb-2 tracking-widest'>Centers of Excellence</h1>
+      <div className='container md:mx-auto md:px-20 md:py-5 m-auto grid grid-flow-row lg:grid-cols-3 md:grid-cols-2 sm:grid-cols-1'>
+        {isCoesLoaded ? getCenters() : noResult()}
```

Comment:
You need to check if getCenters has no data and then show noResult....check if the number of centers in zero. Same goes for all components.